Templating/Helper/BreadcrumbsHelper.php

```diff
@@ -1,79 +1,78 @@
-<?php
-
-namespace WhiteOctober\BreadcrumbsBundle\Templating\Helper;
-
-use Symfony\Component\Templating\Helper\Helper;
-use Twig\Environment;
-use WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs;
-
-class BreadcrumbsHelper extends Helper

```

Comment:
I don't know why there's all this diff, I just deleted the `extend Helper`

Reply:
The file originally had CRLF newlines (Windows ones). I've resolved the problem by adding a separate commit just with the CRLF -> LF conversion